### PR TITLE
Update Hungarian translation

### PR DESCRIPTION
**Pull request by fejese**
Adding missing strings

### CODE REVIEW

**Comment by fejese** [comment]
cc @cortinico as per instructions

File: appintro/src/main/res/values-hu/strings.xml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <string name="app_intro_skip_button">KIHAGYÁS</string>
+    <string name="app_intro_next_button" tools:ignore="MissingTranslation">KÖVETKEZŐ</string>
+    <string name="app_intro_back_button" tools:ignore="MissingTranslation">VISSZA</string>
     <string name="app_intro_done_button">KÉSZ</string>
+    <string name="app_intro_image_content_description" tools:ignore="MissingTranslation">GRAFIKA</string>
```
Comment (cortinico) [inline appintro/src/main/res/values-hu/strings.xml:7]:
```suggestion
    <string name="app_intro_next_button">KÖVETKEZŐ</string>
    <string name="app_intro_back_button">VISSZA</string>
    <string name="app_intro_done_button">KÉSZ</string>
    <string name="app_intro_image_content_description">GRAFIKA</string>
```

**Review by cortinico — CHANGES_REQUESTED** [review]

File: appintro/src/main/res/values-hu/strings.xml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <string name="app_intro_skip_button">KIHAGYÁS</string>
+    <string name="app_intro_next_button" tools:ignore="MissingTranslation">KÖVETKEZŐ</string>
```
Comment (fejese) [inline appintro/src/main/res/values-hu/strings.xml:4]:
```suggestion
    <string name="app_intro_next_button" tools:ignore="MissingTranslation">TOVÁBB</string>
```

File: appintro/src/main/res/values-hu/strings.xml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <string name="app_intro_skip_button">KIHAGYÁS</string>
+    <string name="app_intro_next_button">KÖVETKEZŐ</string>
```
Comment (fejese) [inline appintro/src/main/res/values-hu/strings.xml:4]:
```suggestion
    <string name="app_intro_next_button">TOVVÁBB</string>
```

**Review by fejese — COMMENTED** [review]
Updated PR as per suggestion as well as improved one of the new translations a bit

**Review by cortinico — APPROVED** [review]

**Pull request merged**